mypy_django_plugin/transformers/models.py

```diff
@@ -352,6 +354,25 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
                     if not self.api.final_iteration:
                         raise exc
                     else:
+                        if related_manager_info:
+                            # If a django model has a Manager class that cannot
+                            # be resolved on runtime (like django-money
+                            # monkeypatches all default managers after models
+                            # already have been declared, and generates this
+                            # manager as the return value of a function, so it
+                            # cannot be imported when we try to set up the
+                            # default related managers in
+                            # AddDefaultManagerAttribute), we fallback to a
+                            # default related manager, so you at least get a
+                            # base level of working type checking.
```

Comment:
IMO, involving `django-money` in this comment is a bit too much.

```suggestion
                            # If a django model has a Manager class that cannot
                            # be resolved, we fallback to the default related manager,
                            # so you at least get a base level of working type checking.
```

I mean, if `django-money` changes behaviour this comment becomes obsolete.